proto/osmosis/gamm/v1beta1/genesis.proto

```diff
@@ -42,7 +42,12 @@ message MigrationRecords {
 // be linked to a maximum of one balancer pool.
 message BalancerToConcentratedPoolLink {
   option (gogoproto.equal) = true;
-
-  uint64 balancer_pool_id = 1;
-  uint64 cl_pool_id = 2;
+  PoolID balancerPool = 1 [ (gogoproto.nullable) = false ];
+  PoolID clPool = 2 [ (gogoproto.nullable) = false ];
 }
+
+// used for data serialization
+message PoolID {
+  option (gogoproto.equal) = true;
+  uint64 poolId = 1;
+}
```

Comment:
Can we instead use `sdk.Uint64ToBigEndian` instead of generating this?

I think it will make code cleaner due to not having to initialize `PoolId` struct and being able to use `uint64` directly

Reply:
Forgot for a moment that osmoutils does not support non proto sets, changed as requested